### ISSUE TITLE
Warnings on uninitialized previous values?

### ISSUE THREAD

**Issue by lgwagner** [lgwagner/SpeAR#90]
I find myself using uninitialized previous values quite a bit and I find the warning annoying. Do we still need that validation?

**Comment by AFifarek** [comment]
I concur that the warning is annoying especially when I made the decision
that it is not necessary to define the previous value. It was always more
of a suggestion to the user to consider the need for an initialized
previous value rather than a true warning.

We could do one of three things:

   1. Disable the warning altogether stating that documentation and
   experience should allow the user to determine if the initialized previous
   is necessary.
   2. Enable a preference option to disable that warning if the user would
   like.
   3. Provide an escape in the "previous" grammar construct / new command
   that explicitly shows that user intentionally does not want an initialized
   previous value.

I think option 1, is most appropriate but like some aspects of option 2 and
option 3 especially for users that are not as familiar with the tool.

Bottom line, let's go with option 1 and take a look at the wiki to ensure
it is covered well enough to support that decision.

**Comment by jendavis** [comment]
I'm okay with any of the three options. I lean toward 2 and thought I saw that implemented, but I don't see it in 2.1.0 beta. Lucas, is it in your dev version?
